core/pom.xml

```diff
@@ -301,6 +301,13 @@
                 <version>2.12.4</version>
                 <configuration>
                     <argLine>${surefireArgLine}</argLine> <!-- This is required for code coverage to work. -->
+                    <!-- This is added to reduce log output below 4Meg limit of Travis-CI -->
```

Comment:
removed - will find out if still needed once I push to travis and will add extra PR if needed for just this.